odxtools/determinenumberofitems.py

```diff
@@ -1,18 +1,47 @@
 from dataclasses import dataclass
-from typing import List
+from typing import TYPE_CHECKING, Any, Dict, List, Optional
 from xml.etree import ElementTree
 
-from .odxlink import OdxDocFragment
-from .positioneddataobjectproperty import PositionedDataObjectProperty
-from .utils import dataclass_fields_asdict
+from .dataobjectproperty import DataObjectProperty
+from .exceptions import odxrequire
+from .odxlink import OdxDocFragment, OdxLinkDatabase, OdxLinkId, OdxLinkRef
+
+if TYPE_CHECKING:
+    from ..diaglayer import DiagLayer
 
 
 @dataclass
-class DetermineNumberOfItems(PositionedDataObjectProperty):
+class DetermineNumberOfItems:
```

Comment:
why?

Reply:
40 lines is not "small"

Reply:
it's 38 lines less than `positioneddataobjectproperty.py` ;)